src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -543,7 +550,7 @@ protected function getKernelParameters()
                 'kernel.debug' => $this->debug,
                 'kernel.name' => $this->name,
                 'kernel.cache_dir' => realpath($this->getCacheDir()) ?: $this->getCacheDir(),
-                'kernel.logs_dir' => realpath($this->getLogDir()) ?: $this->getLogDir(),
+                'kernel.logs_dir' => $logDir,
```

Comment:
IMHO, the parameter should not be defined when $hasLogDir is false (see above for what is $hasLogDir).
This won't be a BC break since it's a new feature. And it would prevent wrongly resolving e.g. `%kernel.logs_dir%/foobar` to `/foobar`.